lib/http/request.js

```diff
@@ -165,6 +169,10 @@ module.exports = (function() {
       self.emit('error', {}, response);
     });
 
+    this.request.setTimeout(this.timeoutDuration || 60000, function(){
```

Comment:
could/should we put the default value at the top of request.js as part of the settings (line 9)?

and then we can make the `setTimeout` change the settings like the proxy is set up for example:
```
HttpRequest.setProxy = function(proxy) {
    Settings.proxy = proxy;
  };
```